Dockerfile

```diff
@@ -8,3 +8,4 @@ COPY . .
 
 ENTRYPOINT ["python3"]
 CMD ["/opt/iscsc.fr-notify-bot/main.py"]
+USER blog_bot[:root]
```

Comment:
Yes, because a root image is a docker image running on the root **user**.
The blog_bot user will not have any specific permissions, except it will be part of a specific group (here it will be the root **group**) which will allow the bot to write its logs in a file.

Reply:
ok so not a root image, but just the root group to be able to write logs in a file :ok_hand:

Reply:
@ZynoXelek 

I'd really like the documentation you use to write this line! I never saw such a syntax to define a USER in a Dockerfile.
I also have to remarks:
- you declare it after the `CMD` call, so to me this `USER` will never trigger, this is just like doing something after a `return` statement. But I might be wrong actually! did you test it maybe?
- the `root` group already exists so no worry, but sure the `blog_bot` user don't. However I don't know, maybe Docker automatically creates it, once again a quick test would help for sure!

However seeing [how bitnami does it](https://github.com/bitnami/containers) and in particular this [line](https://github.com/bitnami/containers/blob/74e303ac894b9353e3bab558c8ed7695d52d3d5a/bitnami/mongodb/6.0/debian-11/Dockerfile#L53) this seems to be easier!

Reply:
there have been updates this should be close because it is outdated!